tester/main.cpp

```diff
@@ -263,7 +262,7 @@ int main(int argc, char const *argv[]) {
 	k_tmp.kernel_label = "adi";
 	k_tmp.kernel_folder = "adi";
 	k_tmp.fileName = "adi.c";
-	k_tmp.data_type_option_list = dataType_list_ifd;
+	k_tmp.data_type_option_list = dataType_list_fd;
```

Comment:
Is this the test you mentioned causing errors with integer configuration?

Reply:
Yup, the int datatype brings to a floating point exception in the adi test case, caused by division by zero because when having datatype int, integer conversion is used.